NEW EXTENSION - Offline Time

TheGemDev opened the pull request:
Returns the total time a game is offline or was closed.

<img width="1074" alt="Screenshot 2024-03-08 at 17 18 17" src="https://github.com/GDevelopApp/GDevelop-extensions/assets/53819287/bab9dd83-2a98-4a1b-a6cc-77ac39521c5a">
<img width="1074" alt="Screenshot 2024-03-08 at 17 17 45" src="https://github.com/GDevelopApp/GDevelop-extensions/assets/53819287/b8f65540-8af7-40fd-80af-fa240bd84ce0">
<img width="1074" alt="Screenshot 2024-03-08 at 17 17 43" src="https://github.com/GDevelopApp/GDevelop-extensions/assets/53819287/b7c115d7-89ac-448f-9df2-44af376b95f0">


Project Folder - 
[Archive.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/14540771/Archive.zip)

Extension - 
[Extension.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/14540772/Extension.zip)

TheGemDev commented:
Would be great if you can explain the error.
<img width="828" alt="Screenshot 2024-03-08 at 17 45 41" src="https://github.com/GDevelopApp/GDevelop-extensions/assets/53819287/82999d3d-4f8b-48bd-9839-43e064c24aa7">

Mahan-Ashrafi commented:
i already made it 😄

D8H commented:
Thank you for submitting an extension.

Let's continue talking on the other PR to merge best features of both extensions:
- https://github.com/GDevelopApp/GDevelop-extensions/pull/1093

TheGemDev commented:
i have reworked the extension and it adds the features from  #1093  I have also added the autosave feature. 
<img width="1241" alt="Screenshot 2024-05-13 at 13 24 57" src="https://github.com/GDevelopApp/GDevelop-extensions/assets/53819287/b3acdf14-bd46-49e7-9473-39481d5e1736">

I think the extension should not have a slot to set storage name. Any new storage during the game should overwrite the previous storage. 

It now supports auto save which can be enabled and an interval can be set.

TheGemDev commented:
Project Folder - [offlineTime.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/15294712/offlineTime.zip)
